src/ontology/fbbt-edit.obo

```diff
@@ -127299,14 +127342,14 @@ creation_date: 2010-03-09T11:31:21Z
 
 [Term]
 id: FBbt:00100044
```

Comment:
Given that it is possible that all salt-sensing neurons are bitter-sensing, perhaps add a 'may be equivalent to' annotation linking to FBbt:00049613

Reply:
High salt-sensing neurons in L-type sensilla do not seem to be bitter-sensing.